cardano-crypto-class/src/Cardano/Crypto/PinnedSizedBytes.hs

```diff
@@ -1,24 +1,17 @@
-{-# LANGUAGE BangPatterns               #-}
 {-# LANGUAGE DerivingVia                #-}
 {-# LANGUAGE DataKinds                  #-}
-{-# LANGUAGE GeneralizedNewtypeDeriving #-}
 {-# LANGUAGE KindSignatures             #-}
 {-# LANGUAGE MagicHash                  #-}
 {-# LANGUAGE ScopedTypeVariables        #-}
-{-# LANGUAGE UnboxedTuples              #-}
 {-# LANGUAGE TypeApplications           #-}
 
--- for pinnedByteArrayFromListN
-{-# OPTIONS_GHC -Wno-missing-local-signatures #-}
 module Cardano.Crypto.PinnedSizedBytes
   (
     PinnedSizedBytes,
-    -- * Initialization
-    psbZero,
+    -- * Quasiquoter
+    psbHex,
```

Comment:
Personally, I would not re-export `psbHex` from the main module; most users will *not* need the TH functionality, and for those who do, explicitly importing the `.TH` module won't be prohibitive (and might actually be useful as a reminder that we're using the TH functionality).